src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../Types/todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      {todo.user ? <UserInfo user={todo.user} /> : <span>User not found</span>}
```

Comment:
will be more readable
```suggestion
      {todo.user ? (
        <UserInfo user={todo.user} />
      ) : (
        <span>User not found</span>
      )}
```

Reply:
> will be more readable

I tried to do this, but the linter didn’t let me do it.